.github/workflows/pull_request.yml

```diff
@@ -23,17 +23,19 @@ jobs:
         run: echo "VERSION_SUFFIX=alpha.${{ github.event.number }}" >> $GITHUB_ENV
 
       - name: Setup .NET
-        uses: actions/setup-dotnet@v1.8.2
+        uses: actions/setup-dotnet@v1.8.2
+
+      - run: dotnet restore --verbosity minimal
 
-      - run: dotnet build --configuration ${{ matrix.config }}
+      - run: dotnet build --configuration ${{ matrix.config }} /p:ContinuousIntegrationBuild=true
```

Comment:
An alternative to this change is to put this in the root `Directory.Build.props` and let the `CI` env var provided by the runner turn it on. This is a more generic version of what's described in [clairernovotny/DeterministicBuilds](https://github.com/clairernovotny/DeterministicBuilds) since most build providers set this value.

```xml
<PropertyGroup Condition="'$(CI)' == 'true'">
  <ContinuousIntegrationBuild>true</ContinuousIntegrationBuild>
</PropertyGroup>
```

Reply:
Yep - but i really don't like putting stuff like this in csproj files.

I prefer to just keep it on the command line, when it's requrired.

Reply:
i also *hate* that this is an `msbuild` setting (via the `/p:`)... shit should be first class .. unless i'm missing something?

same with the VERSION of an assembly/nuget

Reply:
![image](https://user-images.githubusercontent.com/899878/141056368-7b14b87b-8ddd-4e06-a7d8-870cb412fd4c.png)